client/payment-details/summary/index.tsx

```diff
@@ -137,7 +137,7 @@ const composePaymentSummaryItems = ( {
 				? riskMappings[ charge.outcome.risk_level ]
 				: '–',
 		},
-	].filter( Boolean );
+	].filter( Boolean ) as HorizontalListItem[];
```

Comment:
I don't like that we're using `as` here, but I'm not sure how to fix it. I've asked in the `#typescript` channel in Slack [here](https://a8c.slack.com/archives/CLCNELZE1/p1688178979998579). Let's see what they say before we merge this 🤔

Reply:
I tried different things, but using `as` was the only thing that worked. 👂I'm listening to the thread.

Reply:
Looks like we can probably use `.filter( isValueTruthy )` via `import { isValueTruthy } from '@automattic/wpcom-checkout';` (see discussion above). I don't see that package available on npmjs.com though, but the function is [small enough](https://github.com/Automattic/wp-calypso/blob/9c924aab383e2e594a5ea85062793a86bc008533/packages/wpcom-checkout/src/is-value-truthy.ts#L1) that it's super easy to just re-implement in the WooPayments codebase.

Reply:
Fixed in [620a60f](https://github.com/Automattic/woocommerce-payments/pull/6249/commits/620a60f3120917305b46628546a6985c896c3c76). Added a comment "Sourced from ..." to acknowledge the source.